civis/futures.py

```diff
@@ -495,35 +484,27 @@ class _ContainerShellExecutor(_CivisExecutor):
     inc_script_names: bool, optional
         If ``True``, a counter will be added to the ``script_name`` to create
         the script names for each submission.
+    **kwargs:
+        Additional keyword arguments will be passed
+        directly to ``scripts.post_containers``.
 
     See Also
     --------
     civis.APIClient.scripts.post_containers
     """
     def __init__(self, docker_image_name="civisanalytics/datascience-python",
                  docker_image_tag="latest",
-                 repo_http_uri=None,
-                 repo_ref=None,
-                 git_credential_id=None,
                  script_name=None,
                  required_resources=None,
```

Comment:
I see your point, but am generally not in favor in any case. Special casing the docker image to latest doesn't do a lot since latest can change under you anyways.

How about we remove the docker image tag one and leave (required resources, docker image name, hidden)?

Then later we remove the `hidden=True` default after we have the archive on success option implemented? This seems like a decent solution.